src/YoloDev.Expecto.TestSdk/settings.fs

```diff
@@ -144,6 +144,8 @@ module RunSettings =
           | true -> Some CLIArguments.Debug
           | false -> None
 
+          "colours", SettingsParser.int >-> CLIArguments.Colours
```

Comment:
Why is this an int? And what are the valid values?

Reply:
It's defined as an int in the Expecto API.

```F#
// Set the level of colours to use. Can be 0, 8 (default) or 256.
    | Colours of int
```

In my testing, it does not throw an error if you set RunSettings.Expecto.Colours to some other value, but if you set     <Colours>8</Colours> or higher, you'll still get the garbled colors in VS output.